classad/__init__.py

```diff
@@ -1,3 +1,54 @@
 """Python package to parse and interpret HTCondor classads"""
+from ._functions import (  # noqa: F401
+    eval,
+    unparse,
+    ifThenElse,
+    isUndefined,
+    isError,
+    isString,
+    isInteger,
+    isReal,
+    isBoolean,
+    int,
+    real,
+    string,
+    floor,
+    ceiling,
+    pow,
+    quantize,
+    round,
+    random,
+    strcat,
+    join,
+    substr,
+    strcmp,
+    stricmp,
+    toUpper,
+    toLower,
+    size,
+    split,
+    splitUserName,
+    splitSlotName,
+    time,
+    formatTime,
+    interval,
+    debug,
+    envV1ToV2,
+    mergeEnvironment,
+    stringListSize,
+    stringListSum,
+    stringListAvg,
+    stringListMin,
+    stringListMax,
+    stringListMember,
+    stringListIMember,
+    stringListsIntersect,
+    regexp,
+    regexps,
+    stringList_regexpMember,
+    userHome,
+    userMap,
+)
+from ._grammar import parse  # noqa: F401
```

Comment:
You should define ``__all__`` if you import things only to export them again.

```python3
__all__ = ["eval", "unparse", ..., "parse"]
```

Reply:
Will do so, thanks :)